notebooks/retrieve_mbox.ipynb

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
<p>I'm not sure we can directly read into the bucket. Downloading locally and saving into the bucket, would be sufficient IMO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>+1 to what @oindrillac said. I don't really see the need for writing directly to s3. But I found this stack-exchnage article that might be helpful if you  want to do this. https://stackoverflow.com/questions/40336918/how-to-write-a-file-or-data-to-an-s3-object-using-boto3/40336919 </p><p> </p><p>Something like:</p><pre class="ql-syntax" spellcheck="false">import boto3

s3 = boto3.resource(
    's3',
    region_name='us-east-1',
    aws_access_key_id=KEY_ID,
    aws_secret_access_key=ACCESS_KEY
)
content="String content to write to a new S3 file"
s3.Object('my-bucket-name', 'newfile.txt').put(Body=content)
</pre><p> </p>